app.js

```diff
@@ -0,0 +1,122 @@
+// Book Class: Represents a Book
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+// Store Class: Handles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
```

Comment:
* This is an OPTIONAL suggestion as it is not a requirement for this project but i think in your spare time you can work on this.
* This method `push` also mutates the array of books you could use other array methods that returns a new array instead.